package.json

```diff
@@ -42,5 +39,12 @@
   },
   "engines": {
     "node": ">=6.1.0"
```

Comment:
Just a question, and I know the final goal of this PR is another topic, but, this node version is ok? Could be this a little annoying according to the new n-gage version?

Reply:
I'm tempted to say that we should be good because `>=` is the equivalent of minVersion (https://docs.npmjs.com/cli/v6/using-npm/semver#usage) but I think it would be wise to pin it to 12. I'll make another PR with just the node version. Excellent spot!